src/TrackArtist.cpp

```diff
@@ -260,6 +260,68 @@ void TrackArtist::UpdatePrefs()
    SetColours(0);
 }
 
+void TrackArt::DrawClipAffordance(wxDC& dc, const wxRect& rect, bool highlight, bool selected, const wxString& title)
+{
+    // bottom rounded corners supposed to be hidden using clipping region or overdraw
+   if (selected)
+   {
+      wxRect strokeRect{
+         rect.x - ClipSelectionStrokeSize,
+         rect.y,
+         rect.width + ClipSelectionStrokeSize * 2,
+         rect.height + ClipFrameRadius };
+      dc.SetBrush(*wxTRANSPARENT_BRUSH);
+      AColor::UseThemeColour(&dc, clrClipAffordanceStroke, clrClipAffordanceStroke);
```

Comment:
Reminder to check whether these and other new colors look good in all four of the built-in themes.  Ask the designers.